### ISSUE TITLE
Stocks.exchange withdrawal

### ISSUE THREAD

**Issue by hellooods** [bitpay/copay#7791]
Has anyone has issues with withdrawing coins to your wallets? Tried to withdraw 2 days ago. It says "approved" but the money has not arrived? How long is the transaction time and is "approved" the final step in the confirmation process? Thanks

**Comment by herveaub** [comment]
yes have the same issue. It's the second time my withdrawal get errors. I emailed their support...let see

**Comment by Gamboster** [comment]
Hello @hellooods and @herveaub , sorry for the delay. Do you guys still having problems?
Please, could you explain better what are you trying to do?
Are you trying to send your funds from the Copay wallet?
Where does "Approved" say?
Where should your funds arrive?
